semantic_segmentation/scripts/train.py

```diff
@@ -24,14 +24,15 @@
 def show_results(imgs, preds, gts):
     fig, ax = plt.subplots(frameon=False)
     fig.set_size_inches(6, 36)
-    colormap = np.array([(16, 64,16), (255,0,0), (0,255,0), (0,0,255),
-                         (255,255,0), (255,0,255), (0,255,255), (255,255,255),
-                         (128,64,16), (128,16,64), (64,16,128), (16,64,128),
-                         (32,64,16), (32,16,64), (64,16,32), (16,64,32)
-                        ], dtype=np.uint8)
+    colormap = np.array([(16, 64, 16), (255, 0, 0), (0, 255, 0), (0, 0, 255),
```

Comment:
in such places I like to put constant number of elements per line e.g. 4 lines with 4 elements
it's easier to check/verify the code quickly

Reply:
black went for 1 per line, so i will leave it this way; 4 per line is too long, it turns out